scapy/utils.py

```diff
@@ -2084,7 +2101,9 @@ def read_packet(self, size=MTU, **kwargs):
         if tshigh is not None:
             p.time = EDecimal((tshigh << 32) + tslow) / tsresol
         p.wirelen = wirelen
-        p.comment = comment
+        p.comments = comments
+        if p.comments is None:
+            p.comment = comment
```

Comment:
I don't get this. I would expect to only set comments (?)

Reply:
This part is included for backward compatibility. If `comments is None`, we assign `comment = comment` to ensure existing usage doesn't break.

Reply:
If I understand it well you're already using a property for `comment` to take the last value of `comments`. this should already take care of compatibility, so I don't understand why it's necessary to put the two.

Reply:
You're right. this isn't necessary anymore, so I'm removing it. Thanks for pointing it out!